tests/Main.hs

```diff
@@ -23,44 +24,28 @@ import qualified TestMake
 import qualified TestUtils
 import qualified TestGraph
 
-import System.IO (hSetEncoding, stdout, stderr, utf8)
+import System.IO (hSetBuffering, stdout, BufferMode(..))
 
 main :: IO ()
 main = do
-  hSetEncoding stdout utf8
```

Comment:
I'm fairly sure this was only necessary in a previous version of the tests because we were previously using `Data.Text.IO.readFile`. We should no longer be using functions like `Data.Text.IO.readFile` which depend on things like locales and encodings associated with particular handles; we should instead be unconditionally treating things as utf-8 by reading them in as ByteStrings and then applying functions like `encodeUtf8` ourselves.